regression-test/suites/show_data_p2/test_table_operation/test_cloud_truncate_table_show_data.groovy

```diff
@@ -0,0 +1,426 @@
+        tableName="test_cloud_truncate_table_normal_show_data"
+        sql "DROP TABLE IF EXISTS ${tableName};"
+        sql """
+            CREATE TABLE IF NOT EXISTS ${tableName}(
+              L_ORDERKEY    INTEGER NOT NULL,
+              L_PARTKEY     INTEGER NOT NULL,
+              L_SUPPKEY     INTEGER NOT NULL,
+              L_LINENUMBER  INTEGER NOT NULL,
+              L_QUANTITY    DECIMAL(15,2) NOT NULL,
+              L_EXTENDEDPRICE  DECIMAL(15,2) NOT NULL,
+              L_DISCOUNT    DECIMAL(15,2) NOT NULL,
+              L_TAX         DECIMAL(15,2) NOT NULL,
+              L_RETURNFLAG  CHAR(1) NOT NULL,
+              L_LINESTATUS  CHAR(1) NOT NULL,
+              L_SHIPDATE    DATE NOT NULL,
+              L_COMMITDATE  DATE NOT NULL,
+              L_RECEIPTDATE DATE NOT NULL,
+              L_SHIPINSTRUCT CHAR(25) NOT NULL,
+              L_SHIPMODE     CHAR(10) NOT NULL,
+              L_COMMENT      VARCHAR(44) NOT NULL
+            )
+            UNIQUE KEY(L_ORDERKEY, L_PARTKEY, L_SUPPKEY, L_LINENUMBER)
+            DISTRIBUTED BY HASH(L_ORDERKEY) BUCKETS 3
+            PROPERTIES (
+              "replication_num" = "1"
+            )
+        """
+        List<String> tablets = get_tablets_from_table(tableName)
+        def loadTimes = [1, 10]
+        Map<String, List> sizeRecords = ["apiSize":[], "mysqlSize":[], "cbsSize":[]]
+        for (int i in loadTimes){
+            // stream load 1 time, record each size
+            repeate_stream_load_same_data(tableName, i)
+            def rows = sql_return_maparray "select count(*) as count from ${tableName};"
+            logger.info("table ${tableName} has ${rows[0]["count"]} rows")
+            // 加一下触发compaction的机制
+            trigger_compaction(tablets)
+
+            // 然后 sleep 5min， 等fe汇报完
+            sleep(300 * 1000)
+
+            sizeRecords["apiSize"].add(caculate_table_data_size_through_api(tablets))
+            sizeRecords["cbsSize"].add(caculate_table_data_size_in_backend_storage(tablets))
+            sizeRecords["mysqlSize"].add(show_table_data_size_through_mysql(tableName))
+            sleep(300 * 1000)
+            logger.info("after ${i} times stream load, mysqlSize is: ${sizeRecords["mysqlSize"][-1]}, apiSize is: ${sizeRecords["apiSize"][-1]}, storageSize is: ${sizeRecords["cbsSize"][-1]}")
+
+        }
+
+        // expect mysqlSize == apiSize == storageSize
+        assertEquals(sizeRecords["mysqlSize"][0], sizeRecords["apiSize"][0])
+        assertEquals(sizeRecords["mysqlSize"][0], sizeRecords["cbsSize"][0])
+        // expect load 1 times ==  load 10 times
+        assertEquals(sizeRecords["mysqlSize"][0], sizeRecords["mysqlSize"][1])
+        assertEquals(sizeRecords["apiSize"][0], sizeRecords["apiSize"][1])
+        assertEquals(sizeRecords["cbsSize"][0], sizeRecords["cbsSize"][1])
+
+        sql """truncate table ${tableName} force;"""
+        sizeRecords["apiSize"].add(caculate_table_data_size_through_api(tablets))
```

Comment:
when force truncate table，use new tablets to check data
and make sure the origin tablets in s3 is deleted